src/libs/antares/study/parts/short-term-storage/series.cpp

```diff
@@ -201,8 +204,8 @@ bool Series::validateInflowsSums(unsigned int firstHourOfTheWeek, unsigned int c
 
         if (sumInjection > sumInflows || sumWithdrawal < sumInflows )
         {
-            logs.warning() << "Error at week: " << firstHourOfTheWeek + 1 << " for sts series "
-                "inflows, sums at cycle timesteps are wrong";
+            logs.warning() << "Error at week: " << firstHourOfTheWeek + 1 << " for short term "
+                "storage inflows, sums at cycle timesteps are wrong";
```

Comment:
The week number is wrong here (it's an hour number). You need to log an hour instead